### PR TITLE
Meal tracking callisto2.0 migration intrepid

### PR DESCRIPTION
**Pull request by LTTS-sugapriya**
Removed the Intrepid library and replaced with the native API's

### CODE REVIEW

File: Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -99,6 +99,15 @@
...
 		93D4F766276CC765FC6209A7 /* Pods_MealTrackingPilotTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F9563EBA0AC2B40CED071536 /* Pods_MealTrackingPilotTests.framework */; };
+		9807AD1323291FC700D56EB9 /* UITextView+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0A23291FC700D56EB9 /* UITextView+Extensions.swift */; };
+		9807AD1423291FC700D56EB9 /* Foundation+Helper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0B23291FC700D56EB9 /* Foundation+Helper.swift */; };
+		9807AD1523291FC700D56EB9 /* CGRect+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0C23291FC700D56EB9 /* CGRect+Extensions.swift */; };
+		9807AD1623291FC700D56EB9 /* UIViewController+Extensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0D23291FC700D56EB9 /* UIViewController+Extensions.swift */; };
+		9807AD1723291FC700D56EB9 /* TypeIdentifiable+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0E23291FC700D56EB9 /* TypeIdentifiable+Extension.swift */; };
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj]: Change the file name to TypeIdentifiable+Extensions.swift

File: Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -99,6 +99,15 @@
...
+		9807AD1823291FC700D56EB9 /* CGPoint+Extension.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9807AD0F23291FC700D56EB9 /* CGPoint+Extension.swift */; };
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj]: Change the file name to CGPoint+Extensions.swift

File: Mobl_ios_Callisto-master/MealTrackingPilot/AppDelegate.swift [hunk 3]
```diff
@@ -21,6 +21,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     let primaryUserStorage = PilotPrimaryUserStorage()
 
+
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot/AppDelegate.swift]: Removed this extra 3 lines of spaces from AppDelegate.

File: Mobl_ios_Callisto-master/MealTrackingPilot/Authentication/LoginFormViewController.swift [hunk 4]
```diff
@@ -146,7 +145,7 @@ class LoginFormViewController: UIViewController, UITextFieldDelegate {
         viewModel.status.subscribe(onNext: { [weak self] (_) in
             guard let welf = self else { return }
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot/Authentication/LoginFormViewController.swift]: Remove this line guard let welf = self else { return }

File: Mobl_ios_Callisto-master/MealTrackingPilot/ChatBot/Views/TypingEllipsisView.swift [hunk 5]
```diff
@@ -71,9 +70,9 @@ class TypingEllipsisView: UIView {
             dot.frame = CGRect(x: 0, y: 0, width: dotDiameter, height: dotDiameter)
             dot.layer.cornerRadius = dotRadius
         }
-        dots[1].center = bounds.ip_center
-        dots[0].center = dots[1].frame.ip_leftMiddle - CGPoint(x: dotSpacing, y: 0)
-        dots[2].center = dots[1].frame.ip_rightMiddle + CGPoint(x: dotSpacing, y: 0)
+        dots[1].center = bounds.cg_center
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot/ChatBot/Views/TypingEllipsisView.swift]: Change the property named from cg_center to center.

File: Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/CGRect+Extensions.swift [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+//  Created by sugapriya on 09/09/19.
+//  Copyright © 2019 Intrepid. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public extension CGRect {
+    
+     var topLeft: CGPoint { return CGPoint(x: minX, y: minY) }
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/CGRect+Extensions.swift]: Please change below properties in correct format like

var topLeft: CGPoint { 
     return CGPoint(x: minX, y: minY) 
}

File: Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/UIView+Constraint.swift [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+
+extension UIView {
+
+    func constrainView(_ view: UIView?, top: CGFloat, left: CGFloat, bottom: CGFloat, right: CGFloat) -> ConstraintDictionary? {
+        return constrainView(view, to: UIEdgeInsets(top: top, left: left, bottom: bottom, right: right))
+    }
+    
+    func constrainView(_ view: UIView?, to insets: UIEdgeInsets) -> ConstraintDictionary? {
+        var constraints: ConstraintDictionary = [:]
+        if insets.top != CGFloat(NSNotFound) {
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/UIView+Constraint.swift:26]: This is not correct way check the insets values in Swift. Can you please check how we will compare this values

File: Mobl_ios_Callisto-master/MealTrackingPilot/Navigation/AppNavigator.swift [hunk 8]
```diff
@@ -29,7 +28,7 @@ final class AppNavigator: NSObject, LoginClientDelegate, LoginClientUIDelegate {
     }()
 
     private lazy var signInViewController: LoginLandingViewController = {
-        let controller = LoginLandingViewController.ip_fromNib()
+        let controller = LoginLandingViewController.init(nibName: LoginLandingViewController.nibName, bundle: nil)
```
Comment (rikky-C291732) [inline Mobl_ios_Callisto-master/MealTrackingPilot/Navigation/AppNavigator.swift]: Please use here the fromNib() instead of whole init.

**Review by rikky-C291732 — COMMENTED** [review]
Please refer the below review comments.

File: Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj (hunk 1 above)
Comment (LTTS-sugapriya) [reply Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj]: Done changes

File: Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj (hunk 2 above)
Comment (LTTS-sugapriya) [reply Mobl_ios_Callisto-master/MealTrackingPilot.xcodeproj/project.pbxproj]: Done changes

File: Mobl_ios_Callisto-master/MealTrackingPilot/AppDelegate.swift (hunk 3 above)
Comment (LTTS-sugapriya) [reply Mobl_ios_Callisto-master/MealTrackingPilot/AppDelegate.swift]: Done changes

File: Mobl_ios_Callisto-master/MealTrackingPilot/Authentication/LoginFormViewController.swift (hunk 4 above)
Comment (LTTS-sugapriya) [reply Mobl_ios_Callisto-master/MealTrackingPilot/Authentication/LoginFormViewController.swift]: done changes

File: Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/CGRect+Extensions.swift (hunk 6 above)
Comment (LTTS-sugapriya) [reply Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/CGRect+Extensions.swift]: done changes

File: Mobl_ios_Callisto-master/MealTrackingPilot/Navigation/AppNavigator.swift (hunk 8 above)
Comment (LTTS-sugapriya) [reply Mobl_ios_Callisto-master/MealTrackingPilot/Navigation/AppNavigator.swift]: done changes

**Review by rikky-C291732 — APPROVED** [review]
Looks good, merge the changes to migration branch.

**Review by rikky-C291732 — APPROVED** [review]

**Pull request merged**